qutip/wigner.py

```diff
@@ -72,7 +72,7 @@ def wigner_transform(psi, j, fullparity, steps, slicearray):
     else:
         rho = psi
 
-    sun = 2   # The order of the SU group
+    sun = 2  # The order of the SU group
```

Comment:
I know it's a bit of a pain, but could you undo the style clean ups from this PR? I would like to add something like black  to QuTiP to make these issues go away, but making all the clean-ups in this PR is asking for trouble (e.g. if something breaks or someone wants to read the history it will be hard for them to see what are the important changes -- indeed it is already a bit tricky for me as a reviewer).